src/components/ControlledImageSearchAndUploader.tsx

```diff
@@ -80,32 +90,19 @@ const ImageSearchAndUploader = props => {
         },
         {
           title: t('form.visualElement.imageUpload'),
-          content: (
+          content: licenses ? (
```

Comment:
Ser ikke helt i farta hvor denne brukes, så vet ikke hvor viktig det er, men burde det være noe annet enn null som returneres hvis licenses er tom? en error plassholder elns?

Reply:
Egentlig så vil det nok aldri skje at du rekker trykke på denne tabben før den har henta licenses. Derfor jeg ikke gjorde noe mer spennende. Dette var bare for å blidgjøre ts egentlig.
Ikke helt sikker på hvor mye logikk man kan legge inn i content feltet i Tabs løsninga.

Reply:
Jeg tenkte først på å ha en spinner eller no, men ombestemte meg fordi jeg var lat